app/routes/listings.routes.js

```diff
@@ -15,4 +15,11 @@ module.exports = function (app) {
         ).post(
             listings.createListing
         );
+
+    app.route('/edit-listing')
```

Comment:
You should change this endpoint to `listings/edit` to better match REST endpoint naming conventions.